minifi-toolkit/minifi-toolkit-configuration/src/test/java/org/apache/nifi/minifi/toolkit/configuration/ConfigMainTest.java

```diff
@@ -173,6 +174,46 @@ public void testTransformRoundTripStressTestFramework() throws IOException, JAXB
         transformRoundTrip("StressTestFramework");
     }
 
+    @Test
```

Comment:
if anticipating a exception, should likely use the @Test(expected = ) in lieu of the try-catch